StudentsApp/Student.cs

```diff
@@ -33,20 +33,20 @@ public Student(string firstName, string lastName, string faculty, int course, in
             this.Group = group;
             this.TasksList = new Queue<string>();
             this.BooksList = new Stack<string>();
-            StudentData = String.Format("{0,15} {1,15} {2,15} {3,15} {4,15}", firstName, lastName, faculty, course, group);
+            this.StudentData = String.Format("{0,15} {1,15} {2,15} {3,15} {4,15}", firstName, lastName, faculty, course, group);
         }
 
         public void GetTasksList()
         {
             Console.WriteLine("Enter tasks. When list is full press spacebar");
-            bool IsListFinshed = false;
+            bool isListFinshed = false;
             string task = string.Empty;
-            while(!IsListFinshed)
+            while(!isListFinshed)
             {
                 task = Console.ReadLine();
                 if (task == " ")
```

Comment:
To fulfill check for whitespace we should use method string.IsNullOrWhiteSpace().

Reply:
But yes, I see you have the similar validation below. OK.